### PR TITLE
Create andsec-buenos-aires-2017.json

### PR DESCRIPTION
**Pull request by MLRiain**

### CODE REVIEW

File: data/events/andsec-buenos-aires-2017.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "event_start": "2017-06-03",
+    "facebook": "https://www.facebook.com/Andseccon/",
+    "hashtag": "#Andsec2017",
+    "languages": [
+        "English", "Spanish"
+    ],
+    "location": {
+        "city": "Buenos Aires",
+        "country": "Argentina",
+        "state": "Buenos Aires"
```
Comment (rollandf) [inline data/events/andsec-buenos-aires-2017.json]: No need of state in this case. Please leave it empty: 
"state": ""

File: data/events/andsec-buenos-aires-2017.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "name": "Andsec Conference",
+    "tags": [
+    	"security"
+    ],
+    "twitter": "andseccon",
+    "website": "http://www.andsec.org/",
+    "youtube": "https://www.youtube.com/watch?time_continue=5&v=AXTB4pQ8jDs"
```
Comment (rollandf) [inline data/events/andsec-buenos-aires-2017.json]: Only the ID of the list is needed:
"youtube": "AXTB4pQ8jDs"

**Comment by MLRiain** [comment]
I do not understand. On this page:
https://github.com/szabgab/codeandtalk.com/blob/main/docs/EVENTS.md
It says:

"state":                "Relevant in US, Brasil, Australia, India, and UK"

So I Googled and found that the state is Buenos Aires. Why is it wrong? Is
the EVENTS.md file wrong or is Buenos Aires state wrong?
This was my first, I would like to get the next one right.

Thank you,

Martha

**Comment by szabgab** [comment]
It is just that *we* only started to collect state information for those 5 countries. (And I think the Brits will not like that we call those states :)

In addition, just an addition to @rollandf I think "AXTB4pQ8jDs" is not even a playlist id so for now I'd leave it out.

**Comment by szabgab** [comment]
much better, thanks
Please also remove the twitter from the missing_events.md in the same branch and push that out too.

**Comment by szabgab** [comment]
thanks

**Pull request merged**